### PR TITLE
WIP: New package: minbif-1.0.5

### PR DESCRIPTION
**Pull request by j0ack**
Minbif package

### CODE REVIEW

File: srcpkgs/minbif/template [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Template file for 'minbif'
+pkgname=minbif
+version=1.0.5
+revision=1
+wrksrc="$pkgname-$version"
+build_style=cmake
+configure_args="PREFIX=/usr CONF_PREFIX=/etc/minbif ENABLE_CACA=$(vopt_if caca 'ON' 'OFF') ENABLE_VIDEO=$(vopt_if caca 'ON' 'OFF') ENABLE_PAM=ON ENABLE_TLS=ON DEBUG=OFF"
```
Comment (maxice8) [inline srcpkgs/minbif/template]: Cut at the last space before hitting 80 columns.

Cmake arguments use `-D<key>=<value>` so you're missind the `-D`

File: srcpkgs/minbif/template [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+build_options="caca"
+desc_option_caca="Build with libcaca support to display icons"
+
+do_configure() {
+	make $configure_args
+}
+
+do_fetch() {
+	git clone https://symlink.me/romain/$pkgname.git $wrksrc
```
Comment (maxice8) [inline srcpkgs/minbif/template]: we **need** a versioned tarball

File: srcpkgs/minbif/template [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+do_install() {
+	make DESTDIR=$DESTDIR install
+}
```
Comment (maxice8) [inline srcpkgs/minbif/template]: missing newline at the end of this newline, see the github UI

File: srcpkgs/minbif/template [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Template file for 'minbif'
+pkgname=minbif
+version=1.0.5
+revision=1
+wrksrc="$pkgname-$version"
```
Comment (maxice8) [inline srcpkgs/minbif/template]: not necessary, this is the default value. wkrsrc= should be set when the value differs from the default.

File: srcpkgs/minbif/template (hunk 1 above)
Comment (j0ack) [reply srcpkgs/minbif/template]: Actually the do_install use `make` which encapsulate the consigure_args without using the `-D`

File: srcpkgs/minbif/template (hunk 2 above)
Comment (j0ack) [reply srcpkgs/minbif/template]: Unfortunatelly I did not found any distfile for this package, I'll try to ask on the minbif IRC channel.

**Comment by maxice8** [comment]
On Fri, Apr 19, 2019 at 04:55:45AM -0700, TROUVERIE Joachim wrote:
> j0ack commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,28 @@
> +# Template file for 'minbif'
> +pkgname=minbif
> +version=1.0.5
> +revision=1
> +wrksrc="$pkgname-$version"
> +build_style=cmake
> +configure_args="PREFIX=/usr CONF_PREFIX=/etc/minbif ENABLE_CACA=$(vopt_if caca 'ON' 'OFF') ENABLE_VIDEO=$(vopt_if caca 'ON' 'OFF') ENABLE_PAM=ON ENABLE_TLS=ON DEBUG=OFF"
> 
> Actually the do_install use `make` which encapsulate the consigure_args without using the `-D`
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/11148#discussion_r276969583

This sounds like a layer of interdiction, can't Cmake be used directly ?

**Comment by maxice8** [comment]
On Fri, Apr 19, 2019 at 04:56:27AM -0700, TROUVERIE Joachim wrote:
> j0ack commented on this pull request.
> 
> 
> 
> > +hostmakedepends="pkg-config git"
> +makedepends="libpurple-devel imlib2-devel gnutls-devel pam-devel $(vopt_if caca libcaca-devel)"
> +short_desc="IRC gateway to IM networks that uses libpurple"
> +maintainer="TROUVERIE Joachim <joachim.trouverie@linoame.fr>"
> +license="GPL-2.0-only"
> +homepage="https://symlink.me/projects/minbif/wiki/"
> +
> +build_options="caca"
> +desc_option_caca="Build with libcaca support to display icons"
> +
> +do_configure() {
> +	make $configure_args
> +}
> +
> +do_fetch() {
> +	git clone https://symlink.me/romain/$pkgname.git $wrksrc
> 
> Unfortunatelly I did not found any distfile for this package, I'll try to ask on the minbif IRC channel.
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/11148#discussion_r276969676

Sorry but if there are no versioned tarballs then there is no way this is included.

**Comment by j0ack** [comment]
> On Fri, Apr 19, 2019 at 04:55:45AM -0700, TROUVERIE Joachim wrote: j0ack commented on this pull request. > @@ -0,0 +1,28 @@ +# Template file for 'minbif' +pkgname=minbif +version=1.0.5 +revision=1 +wrksrc="$pkgname-$version" +build_style=cmake +configure_args="PREFIX=/usr CONF_PREFIX=/etc/minbif ENABLE_CACA=$(vopt_if caca 'ON' 'OFF') ENABLE_VIDEO=$(vopt_if caca 'ON' 'OFF') ENABLE_PAM=ON ENABLE_TLS=ON DEBUG=OFF" Actually the do_install use `make` which encapsulate the consigure_args without using the `-D` -- You are receiving this because you commented. Reply to this email directly or view it on GitHub: [#11148 (comment)](https://github.com/void-linux/void-packages/pull/11148#discussion_r276969583)
> This sounds like a layer of interdiction, can't Cmake be used directly ?

I should be able to use cmake directly I think

**Comment by j0ack** [comment]
> On Fri, Apr 19, 2019 at 04:56:27AM -0700, TROUVERIE Joachim wrote: j0ack commented on this pull request. > +hostmakedepends="pkg-config git" +makedepends="libpurple-devel imlib2-devel gnutls-devel pam-devel $(vopt_if caca libcaca-devel)" +short_desc="IRC gateway to IM networks that uses libpurple" +maintainer="TROUVERIE Joachim ***@***.***>" +license="GPL-2.0-only" +homepage="https://symlink.me/projects/minbif/wiki/" + +build_options="caca" +desc_option_caca="Build with libcaca support to display icons" + +do_configure() { + make $configure_args +} + +do_fetch() { + git clone [https://symlink.me/romain/$pkgname.git](https://symlink.me/romain/%24pkgname.git) $wrksrc Unfortunatelly I did not found any distfile for this package, I'll try to ask on the minbif IRC channel. -- You are receiving this because you commented. Reply to this email directly or view it on GitHub: [#11148 (comment)](https://github.com/void-linux/void-packages/pull/11148#discussion_r276969676)
> Sorry but if there are no versioned tarballs then there is no way this is included.

Ok thanks, I'll ask on the IRC channel.
Unfortunately I tried to get the url from the ARCH repo but it resulted on a 404

**Comment by maxice8** [comment]
On Fri, Apr 19, 2019 at 05:09:23AM -0700, TROUVERIE Joachim wrote:
> > On Fri, Apr 19, 2019 at 04:55:45AM -0700, TROUVERIE Joachim wrote: j0ack commented on this pull request. > @@ -0,0 +1,28 @@ +# Template file for 'minbif' +pkgname=minbif +version=1.0.5 +revision=1 +wrksrc="$pkgname-$version" +build_style=cmake +configure_args="PREFIX=/usr CONF_PREFIX=/etc/minbif ENABLE_CACA=$(vopt_if caca 'ON' 'OFF') ENABLE_VIDEO=$(vopt_if caca 'ON' 'OFF') ENABLE_PAM=ON ENABLE_TLS=ON DEBUG=OFF" Actually the do_install use `make` which encapsulate the consigure_args without using the `-D` -- You are receiving this because you commented. Reply to this email directly or view it on GitHub: [#11148 (comment)](https://github.com/void-linux/void-packages/pull/11148#discussion_r276969583)
> > This sounds like a layer of interdiction, can't Cmake be used directly ?
> 
> I should be able to use cmake directly I think
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/11148#issuecomment-484874534

Please do.

**Pull request closed without merging**